Tests/Functional/SwiftPMXCTestHelperTests.swift

```diff
@@ -0,0 +1,47 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+#if os(OSX)
+
+import XCTest
+import Utility
+
+class SwiftPMXCTestHelperTests: XCTestCase {
+    func testBasicXCTestHelper() {
+        fixture(name: "Miscellaneous/SwiftPMXCTestHelper") { prefix in
+            // Build the package.
+            XCTAssertBuilds(prefix)
+            XCTAssertFileExists(prefix, ".build", "debug", "SwiftPMXCTestHelper.swiftmodule")
+            // Run swift-test on package.
+            XCTAssertSwiftTest(prefix)
```

Comment:
it doesn't work on Linux CI but this tool is anyway for OSX so that bug doesn't affect us for now